static/app/views/starfish/views/webServiceView/endpointOverview/index.tsx

```diff
@@ -178,6 +186,13 @@ export default function EndpointOverview() {
               />
               <Header>
                 <ChartLabel>{DataTitles.throughput}</ChartLabel>
+                <QuestionTooltip
+                  size="sm"
+                  position="right"
+                  title={t(
+                    'the number of requests made to this endpoint per second over a given time range'
```

Comment:
```suggestion
                    'the number of requests made to this endpoint per second in the selected period'
```

can we replace "over a given time range" with "in the selected time range" or "in the selected period" everywhere, it's a little more accurate imo